disable_mobs/data/disable_mobs/functions/install.mcfunction

```diff
@@ -1,6 +1,8 @@
 scoreboard objectives add disabledMobs dummy
 scoreboard objectives add .DisableMob trigger
-scoreboard players set #ONE disabledMobs 1
-scoreboard players set Bat disabledMobs 0
-scoreboard players set Fish disabledMobs 1
-scoreboard players set Guardian disabledMobs 0
+
+execute unless score #ONE disabledMobs matches 1 run scoreboard players set Bat disabledMobs 0
+execute unless score #ONE disabledMobs matches 1 run scoreboard players set Fish disabledMobs 1
+execute unless score #ONE disabledMobs matches 1 run scoreboard players set Guardian disabledMobs 0
+
+scoreboard players set #ONE disabledMobs 1
```

Comment:
```suggestion
execute unless score #INITIALIZED disabledMobs matches 1 run scoreboard players set Bat disabledMobs 0
execute unless score #INITIALIZED disabledMobs matches 1 run scoreboard players set Fish disabledMobs 1
execute unless score #INITIALIZED disabledMobs matches 1 run scoreboard players set Guardian disabledMobs 0

scoreboard players set #INITIALIZED disabledMobs 1
```
I'm assumming this is just used to avoid initializing twice, makes more sense to call it something that makes some reference to that fact. Calling it ONE makes it sound like a constant, where `ONE matches 1` should ALWAYS match